packages/xstate-wallet/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "xstate-wallet",
+  "name": "@statechannels/xstate-wallet",
```

Comment:
This could possibly be a separate PR? Not necessary though.

Reply:
Tempted to leave this change in since it doesn't affect other packages.